submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestFinalizer.kt

```diff
@@ -30,22 +30,29 @@ class SubmissionRequestFinalizer(
         return sub
     }
 
-    private fun deleteRemainingFiles(sub: ExtSubmission, previous: ExtSubmission) {
+    fun deleteSubmissionFiles(submission: ExtSubmission) {
+        deleteRemainingFiles(null, submission)
+    }
+
+    private fun deleteRemainingFiles(current: ExtSubmission?, previous: ExtSubmission) {
```

Comment:
I think this is the logic that should be moved to the `FileStorageService` and then be used in both places since it makes more sense to me to have a storage services that deletes files that later is used by both, the RequestFinalizer and the SubmissionService than the SubmissionService using the RequestFinalizer directly